DOC: add reference to MS Windows pre-release

JensAhrens opened the pull request.

mgeier commented on doc/manual/general.rst:
```diff
@@ -32,13 +32,18 @@ Introduction
+
+The current implementation provides:
+
+- Wave Field Synthesis (WFS)
```
This is unrelated to this PR, but at some point we could provide a link for each of the renderers?
e.g. http://ssr.readthedocs.io/en/latest/renderers.html#wave-field-synthesis-renderer

I think this would be better than a single link to the "renderers" page in the next paragraph.

JensAhrens replied:
I'll add this.

mgeier commented on doc/manual/general.rst:
```diff
@@ -32,13 +32,18 @@ Introduction
+- Wave Field Synthesis (WFS)
+- binaural (HRTF-based) reproduction
+- binaural room (re-)synthesis (BRTF-based reproduction)
+- head-tracked binaural playback
```
I think this renderer ceased to be, right?

JensAhrens replied:
Oh, right, I'll remove this...

JensAhrens commented:
I added cross references in the documentation and I updated a few other details (list of contributors, copyright years, and a reference in the main readme to RTD documentation).
